### PR TITLE
#167454777 create image with article row component

### PR DESCRIPTION
**Pull request by Sojisoyoye**
#### What does this PR do?

This PR sets up image with article component

#### Description of task to be completed?

- setup container for the atoms
- include image atom
- include text and title atom

#### How should this be manually tested?
$ git clone https://github.com/andela/king-kong-ah-frontend.git
$ npm install
$ npm test

#### Any background context you want to provide?

You would need npm to run this application

#### What are the relevant pivotal tracker stories?

PT Story link - #167454777

#### Screenshots (if appropriate)
image

### CODE REVIEW

File: src/components/UI/atoms/Text/Text.jsx [hunk 1]
```diff
@@ -62,6 +64,7 @@ Text.propTypes = {
   content: PropTypes.oneOf(['true', 'false']),
   padding: PropTypes.oneOf(Object.keys(spacing)),
   fontWeight: PropTypes.oneOf(Object.keys(fontWeights)),
+  lineHeight: PropTypes.string,
```
Comment (mosinmiloluwa01) [inline src/components/UI/atoms/Text/Text.jsx]: i think it'll be better if this was one of the properties of styled variable

File: src/components/UI/atoms/Text/Text.jsx [hunk 2]
```diff
@@ -92,6 +96,7 @@ Text.Container = styled.span`
     font-size: ${theme.fontSizes[fontSize]};
     font-weight: ${theme.fontWeights[fontWeight]};
     font-family: ${(content === 'true' && 'Inknut Antiqua') || 'roboto'};
+    line-height: ${lineHeight}
```
Comment (mosinmiloluwa01) [inline src/components/UI/atoms/Text/Text.jsx]: line height should be taken from theme.lineHeights

File: src/components/UI/molecules/ImageText/ImageText.jsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  title,
+  author,
+  description,
+  imageUrl,
+  titleFontSize,
+  authorFontSize,
+  descriptionFontSize,
+}) => {
+  return (
+    <FlexContainer
```
Comment (mosinmiloluwa01) [inline src/components/UI/molecules/ImageText/ImageText.jsx:26]: i'd suggest that you split this component into two smaller components. The first component (i.e image component) has already created by Tosin so you can use that. The second component (ie the article text)should be another component.
Then you will combine the two smaller components into like an `<ArticleSection>` component

File: src/components/pages/Home/HomePage.jsx [hunk 4]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-len */
```
Comment (mosinmiloluwa01) [inline src/components/pages/Home/HomePage.jsx]: please remove this ESLint rule

**Review by mosinmiloluwa01 — CHANGES_REQUESTED** [review]
Good job so far, kindly spare some minutes to view the feedback

File: src/components/UI/atoms/Title/Title.jsx [hunk 5]
```diff
@@ -60,13 +63,15 @@ Title.Container = styled.span`
...
     paddingBottom,
+    content,
     theme,
   }) => `
     display: ${display};
     text-align: ${textAlign};
     color: ${theme.textColors[color]};
     padding-bottom: ${theme.spacing[paddingBottom]};
     font-size: ${theme.fontSizes[fontSize]};
+    font-family: ${(content === 'true' && 'Inknut Antiqua') || 'roboto'};
```
Comment (benfluleck) [inline src/components/UI/atoms/Title/Title.jsx]: Can you try `${content  && 'Inknut Antiqua' || 'roboto'};`

**Review by benfluleck — APPROVED** [review]

**Pull request merged**